src/net/sourceforge/kolmafia/persistence/MallPriceDatabase.java

```diff
@@ -40,7 +40,6 @@
   private static final int CONNECT_TIMEOUT = 15 * 1000;
 
   static {
-    updatePrices("mallprices.txt", false);
     updatePrices("mallprices.txt", true);
```

Comment:
This looks right. I looked at FileUtilities/DataFileUtilities, and this is the argument that allows it to look at the file system, rather than for a built-in file.

It seems like MallPriceDatabase.updatePrices could lose its boolean parameter and always call the lower level method with "true".

Reply:
Or maybe not; there is another updatePrices(filename) which does extra logging and writes data to disk (assuming it had come from the server).

The parameter irritates me, and you get rid of it by renaming a method, but it's not critical.

Reply:
Yeah that's why I didn't remove the parameter I didn't want to merge the functions, I can rename it.